content/en/serverless/google_cloud_run/_index.md

```diff
@@ -2,201 +2,234 @@
+COPY --from=datadog/serverless-init /datadog-init /app/datadog-init
 ENTRYPOINT ["/app/datadog-init"]
 CMD ["/path/to/your-go-binary"] (adapt this line to your needs)
 ```
-See [Tracing Go Applications][1] for more details. [Sample code for a simple Go application][2].
 
+[Sample code for a simple Go application](https://github.com/DataDog/crpb/tree/main/go).
 
-[1]: /tracing/setup_overview/setup/go/?tabs=containers
-[2]: https://github.com/DataDog/crpb/tree/main/go
 {{< /programming-lang >}}
-
 {{< programming-lang lang="python" >}}
 
 ```
-COPY --from=datadog/serverless-init:beta4 /datadog-init /app/datadog-init
+COPY --from=datadog/serverless-init /datadog-init /app/datadog-init
 ENTRYPOINT ["/app/datadog-init"]
 CMD ["ddtrace-run", "python", "app.py"] (adapt this line to your needs)
 ```
 
-See [Tracing Python Applications][1] for detailed instructions. [Sample code for a simple Python application][2].
+[Sample code for a simple Python application](https://github.com/DataDog/crpb/tree/main/python).
 
-[1]: /tracing/setup_overview/setup/python/?tabs=containers
-[2]: https://github.com/DataDog/crpb/tree/main/python
 {{< /programming-lang >}}
-
 {{< programming-lang lang="nodejs" >}}
+
 ```
-COPY --from=datadog/serverless-init:beta4 /datadog-init /app/datadog-init
+COPY --from=datadog/serverless-init /datadog-init /app/datadog-init
 ENTRYPOINT ["/app/datadog-init"]
 CMD ["/nodejs/bin/node", "/path/to/your/app.js"] (adapt this line to your needs)
 
 ```
 
-See [Tracing Node.js Applications][1] for detailed instructions. [Sample code for a simple Node.js application][2].
+[Sample code for a simple Node.js application](https://github.com/DataDog/crpb/tree/main/js).
 
-[1]: /tracing/setup_overview/setup/nodejs/?tabs=containers
-[2]: https://github.com/DataDog/crpb/tree/main/js
 {{< /programming-lang >}}
 {{< programming-lang lang="java" >}}
+
 ```
-COPY --from=datadog/serverless-init:beta4 /datadog-init /app/datadog-init
+COPY --from=datadog/serverless-init /datadog-init /app/datadog-init
 ENTRYPOINT ["/app/datadog-init"]
 CMD ["./mvnw", "spring-boot:run"] (adapt this line to your needs)
 
 ```
 
-See [Tracing Java Applications][1] for detailed instructions. [Sample code for a simple Java application][2].
+[Sample code for a simple Java application](https://github.com/DataDog/crpb/tree/main/java).
 
-[1]: /tracing/setup_overview/setup/java/?tabs=containers
-[2]: https://github.com/DataDog/crpb/tree/main/java
 {{< /programming-lang >}}
-
 {{< programming-lang lang="dotnet" >}}
+
 ```
-COPY --from=datadog/serverless-init:beta4 /datadog-init /app/datadog-init
+COPY --from=datadog/serverless-init /datadog-init /app/datadog-init
 ENTRYPOINT ["/app/datadog-init"]
 CMD ["dotnet", "helloworld.dll"] (adapt this line to your needs)
 
 ```
 
-See [Tracing .NET Applications][1] for detailed instructions. [Sample code for a simple .NET application][2].
+[Sample code for a simple .NET application](https://github.com/DataDog/crpb/tree/main/dotnet).
```

Comment:
Looks like that we don't yet have a .NET sample.

Reply:
Removed the link for now. It can be added back with the example once it's ready